src/Styling/Home.css

```diff
@@ -1,7 +1,75 @@
+/* font styles */
+.style_1 {
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 6.25rem;
+  color: #DE5B31;
+}
+#style_1 {
+  color: transparent;
+  -webkit-text-stroke: 1px #fff;
+}
+.style_2 {
+  font-family: 'Roboto Mono';
+  font-weight: 400;
+  font-size: 17px;
+  color: #fff;
+}
 /* Hero section */
+.Hero_section {
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  position: relative;
+  grid-template-columns: 10% 1fr 2fr 10%;
+  grid-template-rows: 10% 1fr 1fr 10%;
+}
 .Hero_image {
-    width: 100vw;
-    height: 100vh;
-    background-repeat: no-repeat;
-    background-size: cover;
+  width: 100vw;
```

Comment:
Since you’re already setting the width/height for the container of this image above, in “.Hero_section”, you could just do the following here in “.Hero_image”:
width: 100%;
height: 100%;